src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -1106,6 +1542,32 @@ void EIRPlantLoopHeatPump::checkConcurrentOperation(EnergyPlusData &state)
     }
 }
 
```

Comment:
next determines if valid heat recovery HP that is connected to a plant inlet or outlet